client/gutenberg/extensions/videopress/editor.js

```diff
@@ -0,0 +1,25 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import withVideoPressEdit from './edit';
+
+const addVideoPressSupport = ( settings, name ) => {
+	if ( 'core/video' === name ) {
+		const CoreVideoEdit = settings.edit;
+
+		settings = {
+			...settings,
+			edit: withVideoPressEdit( CoreVideoEdit ),
+		};
+	}
+	return settings;
+};
```

Comment:
fun fact! (this is purely style and you don't have to change anything here)

if we want to we can remove the nesting and mutation and replace them with an inline update and ternary expression. why? well, in my mind I like it because it lines up the two branches and I can see all the logic at once. maybe that's not easier for you, but just in case you didn't consider it here's an example

```js
const addVideoPressSupport = ( settings, name ) =>
	'core/video' === name
		? { ...settings, edit: withVideoPressEdit( settings.CoreVideoEdit ) }
		: settings;
```